src/DataFactory/DataFactoryV2/help/Set-AzDataFactoryV2.md

```diff
@@ -134,6 +414,21 @@ Accept pipeline input: True (ByPropertyName)
 Accept wildcard characters: False
 ```
 
+### -TenantId
+The tags of the data factory.
```

Comment:
@sanjkuma this description is incorrect -- what is the tenant id being passed to this cmdlet?

Reply:
Done